janusgraph-test/src/test/java/org/janusgraph/graphdb/tinkerpop/JanusGraphIoRegistryTest.java

```diff
@@ -60,7 +60,7 @@ public void testTinkerPopPredicatesAsGryo() throws SerializationException {
         assertEquals(15,
                 Stream.of(P.class.getDeclaredMethods()).filter(m -> Modifier.isStatic(m.getModifiers())).filter(p -> {
                     log.debug("Predicate: {}", p);
-                    return true;
+                    return !p.isSynthetic();
```

Comment:
JaCoCo instrumentation adds a synthetic method, `$jacocoInit()`, that gets picked up under reflection. See [JaCoCo FAQ](http://www.jacoco.org/jacoco/trunk/doc/faq.html) ("My code uses reflection. Why does it fail when I execute it with JaCoCo?").